.github/workflows/daily.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: 3.9
+          python-version: "3.9"
```

Comment:
Any reason not to bump this to Python 3.10, while we're here?

Reply:
I did at first, but didn't want to muddle the PR. I'm not sure there aren't any side effects.

Reply:
Okay fair, let's leave it for now then